microsoft-edge/webview2/how-to/javascript.md

```diff
@@ -35,10 +35,12 @@ Use the following functions to begin embedding JavaScript in your WebView app.
 ## Scenario: ExecuteScript JSON-encoded results
 
 
-Because the result of `ExecuteScriptAsync` is JSON-encoded, if the result of evaluating the JavaScript is a string, you will receive a JSON-encoded string and not the value of the string. For example, the following script results in a string with the following value, including the quotes at the start and end, and the escaping slashes:
+Because the result of `ExecuteScriptAsync` is JSON-encoded, if the result of evaluating the JavaScript is a string, you will receive a JSON-encoded string and not the value of the string. For example, the following code executes script that results in a string including the quotes at the start and end and the escaping slashes:
```

Comment:
```suggestion
Because the result of `ExecuteScriptAsync` is JSON-encoded, if the result of evaluating the JavaScript is a string, you will receive a JSON-encoded string and not the value of the string. For example, the following code executes script that results in a string.  The resulting string includes a quote at the start, a quote at the end, and escaping slashes:
```